aptos-move/framework/move-stdlib/sources/vector.move

```diff
@@ -589,6 +592,19 @@ module std::vector {
         for_each_reverse(v, |e| d(e))
     }
 
+    public inline fun range(start: u64, end: u64, step: u64): vector<u64> {
```

Comment:
I wonder if it makes sense to have 2 separate range functions - one with step and one without step (default to 1). Also IMO this should not be an inline function as it's long enough and would just bloat people's bytecodes if used multiple times in the same module - this entire code would get expanded a few times.

Reply:
Good point, will remove the `inline` here. I'll create another function named `range_with_single_step`.